src/Compilers/CSharp/Test/Semantic/IOperation/ioperationtests_isymbolinitializer.cs

```diff
@@ -0,0 +1,270 @@
+    Initializer: IObjectCreationExpression (Constructor: Foo..ctor()) (OperationKind.ObjectCreationExpression, Type: Foo)
+        Member Initializers: IPropertyInitializer (Property: System.String Foo.Property1 { get; set; }) (OperationKind.PropertyInitializerInCreation)
+            ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: )");
+
+            compilation.VerifyOperationTree(nodes[3], expectedOperationTree:
+@"IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: Foo x4 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Foo..ctor()) (OperationKind.ObjectCreationExpression, Type: Foo)
+        Member Initializers: IPropertyInitializer (Property: System.String Foo.Property1 { get; set; }) (OperationKind.PropertyInitializerInCreation)
+            ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: )
+          IFieldInitializer (Field: System.Int32 Foo.Field) (OperationKind.FieldInitializerInCreation)
+            ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 2)");
+
+            compilation.VerifyOperationTree(nodes[4], expectedOperationTree:
+@"IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: Foo x5 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Foo..ctor()) (OperationKind.ObjectCreationExpression, Type: Foo)
+        Member Initializers: IPropertyInitializer (Property: Bar Foo.Property2 { get; set; }) (OperationKind.PropertyInitializerInCreation)
+            IObjectCreationExpression (Constructor: Bar..ctor()) (OperationKind.ObjectCreationExpression, Type: Bar)
+              Member Initializers: IFieldInitializer (Field: System.Boolean Bar.Field) (OperationKind.FieldInitializerInCreation)
+                  ILiteralExpression (OperationKind.LiteralExpression, Type: System.Boolean, Constant: True)");
+
+            compilation.VerifyOperationTree(nodes[5], expectedOperationTree:
+@"IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement, IsInvalid)
+  IVariableDeclaration: Foo e1 (OperationKind.VariableDeclaration, IsInvalid)
+    Initializer: IObjectCreationExpression (Constructor: Foo..ctor()) (OperationKind.ObjectCreationExpression, Type: Foo, IsInvalid)
+        Member Initializers: IPropertyInitializer (Property: Bar Foo.Property2 { get; set; }) (OperationKind.PropertyInitializerInCreation, IsInvalid)
+            IConversionExpression (ConversionKind.Invalid, Implicit) (OperationKind.ConversionExpression, Type: Bar, IsInvalid)
+              ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)");
+
+            compilation.VerifyOperationTree(nodes[6], expectedOperationTree:
```

Comment:
This looks to be a bug - the IOperation sub-tree rooted for the bad member initializer `new Foo() { """" };` has no member initializers, but should have an IInvalidExpression with a nested ILiteralExpression (string). I will file an issue tracking this.